portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,12 +42,12 @@
   @Override
   public void init() {
     comments = new ArrayList<>();
-    comments.add(new Comment("Joe", "test1"));
-    comments.add(new Comment("Joe", "test2"));
-    comments.add(new Comment("Joe", "test3"));
-    comments.add(new Comment("Joe", "test4"));
-    comments.add(new Comment("Joe", "test5"));
-    comments.add(new Comment("Joe", "test6"));
+    comments.add(new Comment("Joe", "test1", "email@email.com"));
```

Comment:
Do you not have a data store hooked up to this? I'm surprised we've still got test data.